packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -2674,20 +2681,26 @@ class StackViewport extends Viewport implements IStackViewport {
     this.render();
   }
 
-  private setColormapGPU(colormap: ColormapRegistration) {
+  private setColormapGPU(colormap: ColormapPublic) {
     const ActorEntry = this.getDefaultActor();
     const actor = ActorEntry.actor as ImageActor;
     const actorProp = actor.getProperty();
     const rgbTransferFunction = actorProp.getRGBTransferFunction();
 
+    let colormapObj = colormapUtils.getColormap(colormap.name);
```

Comment:
const colormapObj = colormapUtils.getColormap(colormap.name) || vtkCo....
so that the colormapObj is a constance overall.